managed/devops/bin/yb_backup.py

```diff
@@ -1762,6 +1778,23 @@ def backup_table(self):
         backup location.
         """
 
+        if self.args.storage_type == 'nfs':
```

Comment:
We can't do this check only while backing up. We need to do it for restore as well. We have a function called `post_process_arguments` that we can utilize to do this check before running any commands. Further down the line, we can also expand this post processing to eventually verify the validity of the s3, gcs, and azure credentials.

Reply:
Yup. Make sense. Thanks for catching this! Updated the PR.